packages/vscode-docker-registries/src/clients/RegistryV2/RegistryV2DataProvider.ts

```diff
@@ -124,66 +133,61 @@ export abstract class RegistryV2DataProvider extends CommonRegistryDataProvider
-            throw new Error(vscode.l10n.t('Could not find manifest for tag {0}', item.label));
-        }
-
-        // Parse the embedded JSON strings in the history
-        try {
-            response.body.history?.forEach(history => {
-                history.v1Compatibility = JSON.parse(history.v1Compatibility as unknown as string || '{}');
-            });
-        } catch {
-            // Best effort
-        }
-
+    public async getManifest(item: V2Tag): Promise<unknown> {
+        const response = await this.getManifestResponse(item);
         return response.body;
     }
 
-    protected async getManifestV2(item: V2Tag): Promise<RegistryV2Response<unknown>> {
+    private async getManifestResponse(item: V2Tag): Promise<RegistryV2Response<unknown>> {
         const repository = item.parent as V2Repository;
-        const requestUrl = repository.baseUrl.with({ path: `v2/${item.parent.label}/manifests/${item.label}` });
+        const requestUrl = repository.baseUrl.with({ path: `v2/${repository.label}/manifests/${item.label}` });
 
         return await registryV2Request({
             authenticationProvider: this.getAuthenticationProvider(repository),
             method: 'GET',
             requestUri: requestUrl,
-            scopes: [`repository:${item.parent.label}:pull`],
+            scopes: [`repository:${repository.label}:pull`],
             headers: {
-                'accept': 'application/vnd.docker.distribution.manifest.v2+json,application/vnd.oci.image.index.v1+json'
-            }
+                'accept': acceptManifest.join(','),
+            },
         });
     }
 
-    protected async getTagCreatedDate(item: V2Tag): Promise<Date | undefined> {
-        const manifestv1 = await this.getManifestV1(item);
+    private async getTagCreatedDate(item: V2Tag): Promise<Date | undefined> {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const manifest = await this.getManifest(item) as any;
 
-        const history = manifestv1.history?.[0];
-        return history?.v1Compatibility?.created ? new Date(history.v1Compatibility.created) : undefined;
-    }
-
-    protected abstract getAuthenticationProvider(item: V2RegistryItem): AuthenticationProvider<never>;
-}
+        // Depending on the type of manifest, there's a lot of different places the date might be...
+        if (manifest?.history?.[0]?.v1Compatibility) {
+            // Older v1 manifests have a "history" section with double-encoded JSON
+            const v1Compatibility = JSON.parse(manifest.history[0].v1Compatibility);
+            if (v1Compatibility?.created) {
+                return new Date(v1Compatibility.created);
+            }
+        } else if (manifest?.annotations?.['org.opencontainers.image.created']) {
+            // Some OCI manifests have a date annotation
+            return new Date(manifest.annotations['org.opencontainers.image.created']);
+        } else if (manifest?.config?.digest) {

```

Comment:
This will work if the manifest request comes back with a single manifest, but not if we get a manifest list or OCI index. Technically the depth can be infinite but that probably never happens in practice.

We should chase at least one level deeper--from index -> manifest -> config w/ date. But we cannot allow for infinite depth for obvious reasons.